rfc-095-long-term-future-apps.md

```diff
@@ -0,0 +1,85 @@
+| ckanext-datagovuk | Extension for use with datagovuk_publish | Retire in favour of datagovuk_find/datagovuk_publish? |
+| collections-publisher | Publishes step by steps, /browse pages, and legacy /topic pages on GOV.UK | While topics and browse pages will be removed, this can still be the app for curating collections. We might move Whitehall's document collections into this app |
+| content-audit-tool | Deprecated application to audit content | Retire, as it's not used |
+| content-performance-manager | Data warehouse that stores content and content metrics to help content owners measure and improve content on GOV.UK | Rename to "data warehouse" |
+| content-publisher | WIP - Future publisher of content on GOV.UK | Expand to publish most content |
+| email-alert-service | Message queue consumer that triggers email alerts for GOV.UK | Merge into email-alert-api, as it's only task is to take things from the message queue and forward it there |
+| finder-frontend | Serves search pages for GOV.UK | Rename to "search-frontend". |
+| frontend | Serves the homepage, transactions and some index pages on GOV.UK | Rename to "custom frontend" |
+| licence-finder | Serves licence pages on GOV.UK | Retire |
+| licensify | GOV.UK Licensing (formerly ELMS, Licence Application Tool, & Licensify) | Retire |
+| manuals-frontend | Serves manuals on GOV.UK | Merge into government-frontend |
+| manuals-publisher | Publishes manuals on GOV.UK | Retire in favour of content publisher |
+| organisations-publisher | (Work in progress) Application for managing organisations, people and roles within GOV.UK | Will take on the organisation and "machinery of government" functionality of whitehall |
+| policy-publisher | Publishes policies on GOV.UK | Retire, as the policy format will be retired |
+| publisher | Publishes mainstream content on GOV.UK | Retire in favour of content publisher |
+| router-api | API for updating the routes used by the router on GOV.UK | Merge into either content store or publishing-api |
+| rummager | Search API for GOV.UK | Rename to "search-api" |
+| service-manual-frontend | Serves the Service Manual and Service Toolkit on GOV.UK | Parts of the frontend could be merged into government-frontend, and other apps |
+| service-manual-publisher | Publishes the Service Manual on GOV.UK | Custom application that could long term be replaced by content-publisher  |
+| specialist-publisher | Publishes specialist documents on GOV.UK | Retire in favour of content publisher |
+| static | GOV.UK static files and resources | Retire in favour of [the components gem][cg] |
+| travel-advice-publisher | Publishes foreign travel advice on GOV.UK | Retire in favour of content publisher |
+| whitehall | Publishes government content on GOV.UK | Retire in favour of content publisher, organisations publisher, and collections publisher, custom frontend (history pages) |
+
+[cg]: https://github.com/alphagov/govuk_publishing_components
+
+### No changes expected
```

Comment:
There are a few apps in this section which could be renamed in a similar way to the renames listed in the section above:

- collections to collections-frontend
- feedback to feedback-frontend
- support to support-admin
- smart-answers to smart-answers-frontend

(to name a few)

Should these be added to the section above?